.github/workflows/autoformat.yaml

```diff
@@ -0,0 +1,58 @@
+# https://github.com/peter-evans/create-pull-request/blob/master/docs/examples.md#autopep8
+
+name: auto-protocol-format
+on: pull_request
+jobs:
+  auto-protocol-format:
+    # Check if the PR is not raised by this workflow and is not from a fork
+    if: startsWith(github.head_ref, 'autopep8-patches') == false && github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Setup node 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: install prettier
+        run: npm install -g --save-exact prettier
```

Comment:
you might want to install a given version number, to avoid things suddenly breaking